mail/README.md

```diff
@@ -6,3 +6,51 @@ Start mailhog with `docker-compose up`
  
 You can also add the `mail` service defined in the `docker-compose.yml` file to your projects compose file.
 
+If you do so, be sure to configure the web server with network access to mailhog, such as by adding
```

Comment:
Can just be - " access to mailhog by adding the ..."

Should we call out that it should be added to the build container too if someone wants to see emails that might get triggered by drush commands?